app/services/katello/pulp3/migration.rb

```diff
@@ -47,43 +54,92 @@ def pulp2_repositories_api
       end
 
       def create_and_run_migrations
-        create_migrations.map { |href| start_migration(href) }
+        migs = create_migrations
+        migs.map { |href| start_migration(href) }
+      end
+
+      def self.ignorable_content_types
+        [YumMetadataFile]
       end
 
       def content_types_for_migration
         content_types = @repository_types.collect do |repository_type_label|
           Katello::RepositoryTypeManager.repository_types[repository_type_label].content_types_to_index
         end
 
-        content_types.flatten
+        content_types.flatten - self.ignorable_content_types
```

Comment:
`NoMethodError: undefined method 'ignorable_content_types' for #<Katello::Pulp3::Migration:0x0000000011f82f80>`

Reply:
Works otherwise though if I just hard code in the ignorable content types